### ISSUE TITLE
SASS issue

### ISSUE THREAD

**Issue by SavkaTaras** [lukaszflorczak/vue-agile#82]
SASS is not compiled properly. npm thorws the error during dev. Once deleted the sass from node_modules it seems to be working okay.

**Comment by lukaszflorczak** [comment]
Hi. 

1. Could you show the error?
2. What version of `node.js` do you have?
3. Do you have `sass` in your project (dev) dependencies? What version?

**Comment by SavkaTaras** [comment]
Hi!

Node version 8.9.0.
Node-sass is installed as a part of vue cli set up. (4.11.0 version)


See attached file with logged error.

Thanks,
Taras Savka

**Comment by lukaszflorczak** [comment]
Dependencies in the component package.json:

```json
    "sass": "^1.18.0",
    "sass-loader": "^7.1.0",
```

are also from default project created by vue-cli `3.6.x`. I'll check this.

**Comment by SavkaTaras** [comment]
Thank you, I will check this out.

**Comment by guillecro** [comment]
I confirm that i am having the same error here.. sadly I will have to use another component, but i am glad this was reported

```
Module build failed (from ./node_modules/sass-loader/lib/loader.js):

<template>
^
      Invalid CSS after " ": expected 1 selector or at-rule, was "{"
      in /home/zaqueo/dev/arff-front/node_modules/vue-agile/src/Agile.vue (line 
1, column 1)
```

**Comment by lukaszflorczak** [comment]
Unfortunately, I can't imitate the bug in my environment. Maybe someone with that problem will be able to share with me a repository or though package.json with all dependencies and npm/yarn lock file? It will be very helpful.

**Comment by lukaszflorczak** [comment]
Check `v1.0.5` update and let me know if it helped.

**Comment by SavkaTaras** [comment]
Are you using sass or scss, or did you switch to dart-sass?

Sent from Mail for Windows 10

**Comment by lukaszflorczak** [comment]
@SavkaTaras I updated this comment before you replied. Test also `v1.0.5` release, please.

**Comment by SavkaTaras** [comment]
Thank you. I will try in few hours. Not in from of my laptop now.

Thanks

Sent from Mail for Windows 10

**Comment by SavkaTaras** [comment]
Fixed in 1.0.5.

Thanks!
